src/mds/Capability.h

```diff
@@ -189,6 +189,14 @@ class Capability : public Counter<Capability> {
       _issued = caps;
       // don't add bits
       _pending &= caps;
+
+      // if the revoking is not totally finished, add the new
+      // revoking caps back, or in clean_revoke_from() we may
```

Comment:
The comment may have to remove the reference to `clean_revoke_from()` since it's been removed ?

Reply:
Okay, makes sense. Will fix it. Thanks @kotreshhr